packages/millicast-sdk/src/PeerConnection.js

```diff
@@ -43,13 +43,13 @@ export default class PeerConnection extends EventEmitter {
    * Instance new RTCPeerConnection.
    * @param {RTCConfiguration} config - Peer configuration.
    * @param {Boolean} [autoInitStats = true] - True to initialize statistics monitoring of the RTCPeerConnection accessed via Logger.get(), false to opt-out.
+   * @param {Number} [statsIntervalMs = 1000] - The default interval at which the SDK will return WebRTC stats to the consuming application
    */
-  async createRTCPeer (config = { autoInitStats: true }) {
-    logger.info('Creating new RTCPeerConnection')
-    logger.debug('RTC configuration provided by user: ', config)
+  async createRTCPeer (config = { autoInitStats: true, statsIntervalMs: 1000 }) {
+    logger.info('Creating new RTCPeerConnection', config)
     this.peer = instanceRTCPeerConnection(this, config)
     if (config.autoInitStats) {
-      this.initStats()
+      this.initStats(config.autoInitStats, config.statsIntervalMs)
```

Comment:
Couldn't we pass in config object and then extract only the relevant options. Mainly thinking about scalability

Reply:
Yes we can and now we do :)